docs/content/docs/deployment/elastic_scaling.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Elastic Scaling
+weight: 5
+type: docs
+
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Elastic Scaling
+
+Flink allows you to adjust your cluster size dynamically to your workloads. This is possible by stopping a job with a savepoint and restarting it with a different parallelism. This page describes options where Flink automatically adjusts the parallelism.
+
+## Reactive Mode
+
+{{< hint danger >}}
+Reactive mode is an experimental feature. The Flink community is actively looking for feedback by users through our mailing lists.
+{{< /hint >}}
+
```

Comment:
I think it would be good to explain on a high level how the reactive mode works: E.g. Flink tries to make use of as many resources as possible by adapting the job as the set of resources changes. Then it is easier to understand how this mode can be used to build an autoscaling mechanism.